iceoryx_hoofs/test/moduletests/test_concurrent_sofi.cpp

```diff
@@ -328,15 +331,15 @@ TEST_F(CUnitTestContainerSoFi, MultiOverflow)
 TEST_F(CUnitTestContainerSoFi, ResizeFailsWhenContainingASingleElement)
 {
     ::testing::Test::RecordProperty("TEST_ID", "9c7c43d8-939c-4fa8-b1b9-b379515931e9");
-    int ret;
+    int ret{-1};
     m_sofi.push(123, ret);
     EXPECT_EQ(m_sofi.setCapacity(4), false);
 }
 
 TEST_F(CUnitTestContainerSoFi, ResizeFailsWhenContainingAMultipleElements)
 {
     ::testing::Test::RecordProperty("TEST_ID", "a98bd656-7d39-4274-a77f-bc918a2c1301");
-    int ret;
+    int ret{-1};
```

Comment:
I was constantly wondering why `ret{-1}` until I realized it has no meaning at all.

What do you think of removing `ret` and turn it into a member inside the `class CUnitTestContainerSoFi` and initialize it there with `0`? Zero is more intuitive then `-1`, with `-1` one suspects always a deeper meaning ... or at least I am.

Reply:
There is always a deeper meaning ;) I set `ret` to `-1` because this is a value which should not be present in the test SoFi and in some tests this value is explicitly checked.